app/assets/stylesheets/ontologies.scss

```diff
@@ -2,6 +2,10 @@ $ont-metadata-bg-color: #e2ebf0;
 $widget-table-border-color: #EFEFEF;
 
 
+.resolvability-check svg path{
```

Comment:
If we merge the PR this need to be deleted

Reply:
this was also required by https://github.com/biodivportal/ontoportal_web_ui/issues/9

Reply:
Yes in our case we don't want it

Reply:
OK dine in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/748/commits/af478b44b4efbe3b4bc32ebd146af744143b8766

Reply:
Perfect, I'll merge it then